.env

```diff
@@ -0,0 +1 @@
+MONGOHQ_URL=mongodb://localhost:27017/cfh
```

Comment:
Delete this file from git and add it to .gitignore.

Reply:
done